index.html

```diff
@@ -15,54 +16,84 @@
 </head>
 
 <body>
```

Comment:
I don't see a selector for your `<body>` in your LESS files, Maybe i missed it, but good practice would be to see whether you `<body>` actually encloses your whole HTML content in your browser (coloring a border would help to just make sure)